augly/tests/video_tests/transforms/composite_test.py

```diff
@@ -74,11 +83,12 @@ def test_RandomPixelization(self):
     def test_ReplaceWithBackground(self):
         self.evaluate_class(
             vidaugs.ReplaceWithBackground(
-                source_offset=0.1,
-                background_offset=0,
+                source_offset=0.1, 
+                background_offset=0, 
```

Comment:
+1 for added whitespace
```suggestion
                source_offset=0.1,
                background_offset=0,
```